src/notes/notes.js

```diff
@@ -152,14 +120,11 @@ function _deleteCurrentNote(event) {
     // this allows us to have _deleteCurrentNote be a function that doesn't take in any parameters
     // so that we can use it with addEventListener and deleteEventListener to add/remove it.
     setTimeout(() => {
-        _deleteFromStorage(CURRENT_NOTE_ID);
+        deleteFromStorage(CURRENT_NOTE_ID);
     }, 10);
 }
 
-/**Edit the current note (displays note editor popup)
- * To be used with add/remove event listeners
```

Comment:
Why were these comments removed? Are these not of JSDocs format?

Reply:
Added this back in. Will change the rest of the comments to JSDocs format in a new branch `cleanup-notes`